### DISCUSSION TITLE
keep order of user selection?

### DISCUSSION THREAD

**Discussion by mostafa-norouzi** [brianvoe/slim-select#484] [Ideas]
Hi

Is it possible that plugin keep order of user selection?
It seems that already slim-select re-order them.

With the best.

**Comment by artifex-media** [comment]
+1

I really need this

**Comment by brianvoe** [comment]
I built it for you guys. the field will be keepOrder. It will go out in the next release.

Please consider sponsoring https://github.com/sponsors/brianvoe?frequency=one-time&sponsor=brianvoe
